test/JIT_Test/playlist.xml

```diff
@@ -217,6 +217,7 @@
 	mathSimplifyTest,\
 	NewInstanceTest,\
 	signExtensionATest,\
+	StringPeepholeTest,\
```

Comment:
this is added to a suite that is run with "-Xjit:noJitUntilMain -Xdump:java" as the commandline options, is that sufficient for your test?  You mentioned a few others in our conversation "-Xjit:{*theMethodWithCodePatternfortesting*}(count=0,optlevel=hot),dontinline={*theMethodWithCodePatternfortesting*}"